apps/api/src/app/auth/services/auth.service.ts

```diff
@@ -1,12 +1,7 @@
 import { Inject, Injectable } from '@nestjs/common';
 import { MemberEntity, SubscriberEntity, UserEntity } from '@novu/dal';
-import {
-  AuthProviderEnum,
-  AuthenticateContext,
-  ISubscriberJwt,
-  UserSessionData,
-} from '@novu/shared';
-import { IAuthService } from './auth.service.interface';
+import { AuthProviderEnum, AuthenticateContext, ISubscriberJwt, UserSessionData } from '@novu/shared';
```

Comment:
Do we need these to remain in shared?

Reply:
Some of these likely need to stay e.g. `ISubscriberJwt` is used in `@novu/ws`, but I would assume atleast half of the types can probably be also moved to API - we can revise types in other PRs.